src/ol/interaction/Draw.js

```diff
@@ -630,7 +641,7 @@ class Draw extends PointerInteraction {
       if (this.shouldHandle_) {
         if (!this.finishCoordinate_) {
           this.startDrawing_(event);
-          if (this.mode_ === Mode.POINT) {
+          if (this.mode_ === Mode.POINT || this.maxPoints_ < 2) {
```

Comment:
Instead of deciding this based on `this.maxPoints_`, The `finishCondition` should be tested here, like for the other geometry types.

Reply:
When `maxPoints` is 1 that test makes the behaviour exactly the same as a single Point draw (and the current MultiPoint).   How could a `finishCondition` be relevant it that situation?